capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -147,7 +149,13 @@ function displayCollegeSuggestions(suggestions, autocompleteList, value) {
     const copyArrayElement = arrayElement;
     listElement.addEventListener('click', () => {
       SEARCH_INPUT.value = copyArrayElement;
-      keepTrackOfChosenColleges(copyArrayElement);
+      if (document.getElementById(copyArrayElement.collegeId.id)) {
```

Comment:
I don't understand why we need this `if`. Are you worried that the click event might get triggered after the list element is removed?

Reply:
no, it's to address the instance where 'Stanford University' has already been selected and the user tries to reselect 'Stanford University'